tl/cli/get-kg-links.py

```diff
@@ -29,7 +29,7 @@ def add_arguments(parser):
                         default=5,
                         help='desired number of output candidates per input cell.')
 
-    parser.add_argument('--k-rows', action='store_true', type=bool, dest='k_rows', required=False,
+    parser.add_argument('--k-rows', action='store', type=bool, dest='k_rows', required=False,
```

Comment:
why did you change from store_true ? @rijulvohra

Reply:
@saggu It was giving the follwing error on running the command `tl get-kg-links -h`
<img width="711" alt="Screen Shot 2021-04-13 at 12 05 23 PM" src="https://user-images.githubusercontent.com/19925199/114606880-b4d35e80-9c50-11eb-9609-c649fbf020d8.png">

Reply:
ah, so remove the type parameter from the parameter definition